src/olympia/devhub/templates/devhub/versions/list.html

```diff
@@ -212,7 +221,7 @@ <h3>{{ _('All versions') }}</h3>
   <div class="item" id="version-list"
        data-stats="{{ url('devhub.versions.stats', addon.slug) }}">
     <div class="item_wrapper">
-      {% if check_addon_ownership(request.user, addon, allow_developer=True) and addon.status != amo.STATUS_DISABLED %}{# i.e. Admin disabled #}
+      {% if can_submit %}
```

Comment:
As above, since the view is already decorated with the appropriate check and `can_submit` already means the add-on is not disabled, we can simplify this.